libs/design-system/cards/content-button/src/lib/content-button.component.ts

```diff
@@ -17,8 +17,8 @@ import { ButtonsModule } from '@hra-ui/design-system/buttons';
 export class ContentButtonComponent {
   /** Image url */
   readonly imageSrc = input.required<string>();
-  /** Date to display on card */
-  readonly date = input.required<string>();
+  /** Date timestamp */
+  readonly date = input.required<number>();
```

Comment:
Maybe we change this to `string | number` to allow both timestamps and date strings